src/components/Machine/StarterMachine/StarterMachine.jsx

```diff
@@ -25,11 +25,11 @@ class StarterMachine extends Component {
   }
 
   onClick = () => {
-    this.setState(state => ({ open: !state.open }), this.updateSelection)
+    this.setState(state => ({ ...state, open: !state.open }))
   }
 
   onMaterialSelected = material => () => {
-    this.setState({ materialSelected: material }, this.updateSelection)
+    this.setState(() => ({ open: false, materialSelected: material }), this.updateSelection)
```

Comment:
I would like that the popUp won't be closed when you select a material, i think that, if you want create more than a starter machine, for example, you want create a gold machine, a silver machine and 2 iron machines then you will have to do click once time for each type change, what do you think ?

Reply:
err, what? Should a starter machine create more than a single material?